### PR TITLE
Update to add change in certificate selection logic

### PR DESCRIPTION
**Pull request by amenarde**

### CODE REVIEW

**Review by peterpogorski — APPROVED** [review]

File: release_notes/Service-Fabric-72CU4.md [hunk 1]
```diff
@@ -32,6 +32,22 @@ Microsoft Azure Service Fabric 7.2 Fourth Refresh Release Notes
 
 - Service Fabric 7.2 and higher runtime drops support for .NET core Service Fabric apps running with .NET core 2.2 runtime. .NET core runtime 2.2 is out of support from Dec 2019. Service Fabric runtime will not install .NET core runtime 2.2 as part of its dependency. Customers should upgrade their .NET 2.2 runtime SF apps to the next .NET core LTS version 3.1.
 
+Starting with Service Fabric 7.2. CU4, the logic the runtime uses to select the Cluster certificate has changed. This change applies to both clusters with certificates declared by common name, and clusters with certificates declared by thumbprint, which also have a secondary thumbprint declared.
```
Comment (dragav) [inline release_notes/Service-Fabric-72CU4.md:35]: >Starting [](start = 0, length = 8)

continue with the enumeration: "- Service Fabric 7.2 CU4..."

Comment (dragav) [inline release_notes/Service-Fabric-72CU4.md:35]: >. [](start = 32, length = 1)

please remove (the period after 7.2.)

Comment (dragav) [inline release_notes/Service-Fabric-72CU4.md:35]: >the logic the runtime uses to select the Cluster certificate has changed. This change applies to both clusters with certificates declared by common name, and clusters with certificates declared by thumbprint, which also have a secondary thumbprint declared. [](start = 39, length = 257)

..the runtime will select the certificate with the largest NotBefore value (the most recently issued), when choosing from multiple matches; previously the runtime selected the certificate with the largest NotAfter value (the farthest-expiring) from multiple matches. This change applies to both certificates declared by common name, and those declared by thumbprint when the Security/UseSecondaryIfNewer setting is set to true.

File: release_notes/Service-Fabric-72CU4.md [hunk 2]
```diff
@@ -32,6 +32,22 @@ Microsoft Azure Service Fabric 7.2 Fourth Refresh Release Notes
...
+Example
+| Certificate | Thumbprint | NotBefore | NotAfter (expiry) |
+| - | - | - | - |
+| Primary | AAAA... | 1/1/2020 | 1/1/2023 |
+| Secondary | BBBB... | 6/1/2020 | 6/1/2021 |
+
+In SF versions before 7.2. CU4, SF will select AAAA..., because among the two time-valid certificates, it is the furthest living certificate (1/1/2023). Starting in 7.2. CU4, SF will select BBBB..., because among the two, it is the newer certificate (6/1/2020).
+
+Similarly, for clusters with certificates declared by common name, the selected cluster certificate will change if among the certificates with the declared common name, the longest-living certificate is not also the newest certificate.
+
```
Comment (dragav) [inline release_notes/Service-Fabric-72CU4.md:48]: Please remove; good content, best suited for the main doc (which we should also update).

**Review by dragav — APPROVED** [review]
:shipit:

**Pull request merged**